tests/test-vip-plugins.php

```diff
@@ -92,7 +92,7 @@ public function test__modify_active_plugins() {
 		/**
 		 * Check that the returned option matches a merge of the filtered loaded plugins and active plugins
 		 */
-		$merged_plugins = array_merge( $this->option_active_plugins, wpcom_vip_get_filtered_loaded_plugins() );
+		$merged_plugins = array_unique( array_merge( $this->option_active_plugins, wpcom_vip_get_filtered_loaded_plugins() ) );
```

Comment:
The `array_unique` here feels like a patch for failing tests, where dupes exists. It might be covering some bugs in the tests. The `array_unique` should be removed once the tests get refactored and broken down to individual cases.